modules/apps/asset/asset-browser-web/src/main/java/com/liferay/asset/browser/web/internal/display/context/AssetBrowserDisplayContext.java

```diff
@@ -578,6 +587,9 @@ private BreadcrumbEntry _getSitesAndLibrariesBreadcrumb()
 					PortalUtil.getLiferayPortletResponse(_renderResponse))
 			).setParameter(
 				"groupType", "site"
+			).setParameter(
+				"scopeGroupType",
+				ParamUtil.getString(_httpServletRequest, "scopeGroupType", "")
```

Comment:
The default value is redundant; it will return StringPool.BLANK if the parameter has no value. Can you remove it?